ui/src/translations/fr.json

```diff
@@ -108,5 +108,8 @@
   "volume_statue_failed_pv_bound_hint": "Le volume est lié à une demande de stockage.",
   "delete_a_volume": "Supprimer un volume",
   "delete_a_volume_warning": "La suppression de ce volume entraînera la perte des données qu'il contient.",
-  "delete_a_volume_confirm": "Voulez-vous supprimez "
+  "delete_a_volume_confirm": "Voulez-vous supprimez ",
+  "select_a_storageClass": "Sélectionner une classe de stockage...",
+  "select_a_type": "Sélectionner un type...",
+  "not_found": "Pas trouvé"
```

Comment:
`Pas trouvé` is really weird.
`Aucun résultat` would be better in this case (and to be consistent, use `No results` for the English translation + rename `not_found` to `no_results`.